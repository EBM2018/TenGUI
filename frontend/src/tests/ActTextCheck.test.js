import ActTextCheck from '../react/Widgets/Activity/ActTextCheck';
import React from 'react';
import renderer from 'react-test-renderer';
test ('derp',()=>{
    expect('True').toBeTruthy;
});
