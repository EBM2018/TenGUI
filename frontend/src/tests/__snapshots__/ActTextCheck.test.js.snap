// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does this render 1`] = `
<div
  id="container reponse"
>
  <div>
     
    maybe
     
  </div>
  <input
    onChange={[Function]}
    type="checkbox"
  />
  <label>
     
     
  </label>
  <input
    onChange={[Function]}
    type="checkbox"
  />
  <label>
     
     
  </label>
</div>
`;
